app/views/insured/group_selection/_cancel_plan_form.html.erb

```diff
@@ -2,13 +2,17 @@
   <div class="confirmation hidden" id="cancel-plan-form">
     <h2><%= l10n("plan_cancelation") %></h2>
     <p id="cancel-msg" class="action-msg hidden"><%= l10n("cancel_plan_confirmation") %></p>
-    <label class="required" for="cancel_plan"> <%= l10n("cancel_your_plan") %></label>
+    <%= label_tag :cancel_plan, l10n("cancel_your_plan"), class: "required" %>
     <div class="row focus">
       <div class="col-auto">
-        <label for="cancel_plan_yes"><%= radio_button_tag :cancel_plan, "yes" %>&nbsp;<%= l10n("yes") %></label><br>
+        <%= label_tag :cancel_plan_yes do %>
+          <%= radio_button_tag :cancel_plan, "yes" %>&nbsp;<%= l10n("yes") %>
```

Comment:
rather than spaces, add a class to the radio button of mr-2 (or mr-1, use your best judgement on the spacing)